src/components/pages/PendingApplications/ApplicationModal.js

```diff
@@ -69,7 +69,7 @@ const ApplicationModal = ({
       axiosWithAuth()
         .get(`/application/profileId/${profileId}`)
         .then(res => {
-          setCurrentApplication(res.data);
+          setCurrentApplication(res.data[1]);
```

Comment:
setting current application to a specific entry rather than the whole array?

Reply:
We set this up temporarily so that we could pull some actual data in order to complete the button styling. It wasn't pulling any data with the previous setup. This will be modified in the next update.